src/css/slider.scss

```diff
@@ -11,6 +11,12 @@
     }
   }
 
+  &__arrows {
```

Comment:
n da pra adc classe com computada, no caso do proprio quasar?

Reply:
adicionei uma computada hideArrows, utilizando this.$q.platform.is.mobile